docs/codecov.md

```diff
@@ -0,0 +1,54 @@
+# New Relic PHP Agent code coverage monitoring
+
+New Relic PHP Agent's CI/CD is integrated with [Codecov](https://about.codecov.io/)
+to provide insights into agent's automated tests coverage. Insights can be accessed
+through via Codecov's dashboard for [`newrelic-php-agent`](https://app.codecov.io/gh/newrelic/newrelic-php-agent).
+
+## Codecov's dashboard for [`newrelic-php-agent`](https://app.codecov.io/gh/newrelic/newrelic-php-agent)
+
+Codecov's dashboard for [`newrelic-php-agent`](https://app.codecov.io/gh/newrelic/newrelic-php-agent) has many
+useful views. Here's a list of most useful ones:
+* [Coverage by PHP version](https://app.codecov.io/gh/newrelic/newrelic-php-agent/flags)
+* [Coverage by pull requests](https://app.codecov.io/gh/newrelic/newrelic-php-agent/pulls)
+* [Coverage by commit by branch](https://app.codecov.io/gh/newrelic/newrelic-php-agent/commits)
+* [Coverage by file by branch](https://app.codecov.io/gh/newrelic/newrelic-php-agent)
+
+## New Relic PHP Agent's CI/CD integration with [Codecov](https://about.codecov.io/)
+
+New Relic PHP Agent's CI/CD integration with [Codecov](https://about.codecov.io/)
```

Comment:
Duplicate of the line above?

Reply:
No. The line with `##` is section title. The line without `##` is first line of the paragraph in this section. I think it was more obvious when the paragraph was formatted without numbered list - see [previous version](https://github.com/newrelic/newrelic-php-agent/blob/c6e572682c3b2624aaa800c4e16f362e36fe1a83/docs/codecov.md#new-relic-php-agents-cicd-integration-with-codecov). Now I think `:` is in order at the end of the line, like this:
```suggestion
New Relic PHP Agent's CI/CD integration with [Codecov](https://about.codecov.io/):
```